src/utils/utils.cairo

```diff
@@ -92,36 +91,36 @@ namespace Helpers {
         let res = Uint256(low=low, high=high);
         return res;
     }
-    // @notice This function is used to convert a sequence of i bytes to Uint256.
-    // @param val: pointer to the first byte.
-    // @param i: sequence size.
+    // @notice This function is used to convert bytes array in big-endian to Uint256.
+    // @param bytes_len: bytes array length.
+    // @param bytes: pointer to the first byte of the bytes array.
     // @return res: Uint256 representation of the given input in bytes.
-    func bytes_i_to_uint256{range_check_ptr}(val: felt*, i: felt) -> Uint256 {
+    func bytes_big_endian_to_uint256{range_check_ptr}(bytes_len: felt, bytes: felt*) -> Uint256 {
         alloc_locals;
 
-        if (i == 0) {
+        if (bytes_len == 0) {
             let res = Uint256(0, 0);
             return res;
         }
 
-        let is_sequence_32_bytes_or_less = is_le(i, 32);
+        let is_bytes_len_32_bytes_or_less = is_le(bytes_len, 32);
         with_attr error_message("number must be shorter than 32 bytes") {
-            assert is_sequence_32_bytes_or_less = 1;
+            assert is_bytes_len_32_bytes_or_less = 1;
         }
 
-        let is_sequence_16_bytes_or_less = is_le(i, 16);
+        let is_bytes_len_16_bytes_or_less = is_le(bytes_len, 16);
 
         // 1 - 16 bytes
-        if (is_sequence_16_bytes_or_less != FALSE) {
-            let (low) = compute_half_uint256(val=val, i=i, res=0);
+        if (is_bytes_len_16_bytes_or_less == TRUE) {
+            let (low) = compute_half_uint256_from_bytes(bytes_len, bytes, 0);
```

Comment:
let's just use `bytes_to_felt` function and remove the `compute_half_uint256`

Reply:
Done